tcms/bugs/tests/test_permissions.py

```diff
@@ -138,6 +138,18 @@ def verify_post_with_permission(self):
         self.assertEqual(comments.last().comment, 'A comment text')
 
 
+class TestSearch(tests.PermissionsTestCase):
+    permission_label = 'bugs.view_bug'
+    url = reverse('bugs-search')
+    http_method_names = ['get']
+    post_data = 1   # attribute is still checked even when POST isn't in http_method_names
```

Comment:
See #1915 & rebase after it has been merged.

Reply:
Okay.